xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp

```diff
@@ -203,48 +197,47 @@ bool CGUIDialogPVRGuideInfo::OnMessage(CGUIMessage& message)
   return CGUIDialog::OnMessage(message);
 }
 
-void CGUIDialogPVRGuideInfo::SetProgInfo(const CFileItem *item)
+void CGUIDialogPVRGuideInfo::SetProgInfo(const EPG::CEpgInfoTagPtr &tag)
 {
-  *m_progItem = *item;
+  m_progItem = tag;
```

Comment:
Ah, and the point is if it was changed/passed around this would be correct. Having no copies  anymore but acting on single shared instances is a design principle for pvr data.